packages/configs/tsc/index.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@parcel/config-default",
+  "transforms": {
+    "*.{ts,tsx}": [
+      "@parcel/transformer-typescript-tsc",
+      "@parcel/transformer-js",
```

Comment:
should we also run babel here? Might be surprising for TS users to always have their code run through babel, though it would be limited to `preset-env`, `flow-strip-types` (unnecessarily), and `preset-react` (unnecessarily)

Reply:
Thought we only added `flow-strip-types` if we saw '// @flow' and something similar for React? `babel-preset-env` seems like it would still be valuable unless tsc is capable of something similar.